src/pages/es/reference/configuration-reference.mdx

```diff
@@ -465,22 +486,41 @@ import rehypeMinifyHtml from 'rehype-minify';
 ```
 
 
-### markdown.extendDefaultPlugins
+### markdown.gfm
 
 <p>
 
 **Tipo:** `boolean`<br />
-**Por defecto:** `false`
+**Por defecto:** `true`<br />
+<Since v="2.0.0" />
 </p>
 
-Astro aplica los plugins [GitHub-flavored Markdown](https://github.github.com/gfm/) y [Smartypants](https://github.com/silvenon/remark-smartypants) por defecto. Si deseas añadir tus propios plugins de remark o rehype, puedes mantener los incluidos por defecto, puedes hacerlo estableciendo la flag `extendDefaultPlugins` con el valor `true`:
+Astro usa [GitHub-flavored Markdown](https://github.com/remarkjs/remark-gfm) por defecto. Para deshabilitarlo, establece la flag `gfm` a `false`:
 
 ```js
 {
   markdown: {
-    extendDefaultPlugins: true,
-    remarkPlugins: [exampleRemarkPlugin],
-    rehypePlugins: [exampleRehypePlugin],
+    gfm: false,
+  }
+}
+```
+
+
+### markdown.smartypants
+
+<p>
+
+**Type:** `boolean`<br />
+**Default:** `true`<br />
+<Since v="2.0.0" />
+</p>
+
+Astro uses the [SmartyPants formatter](https://daringfireball.net/projects/smartypants/) by default. To disable this, set the `smartypants` flag to `false`:
```

Comment:
```suggestion
**Tipo:** `boolean`<br />
**Por defecto:** `true`<br />
<Since v="2.0.0" />
</p>

Astro usa el formateador [SmartyPants](https://daringfireball.net/projects/smartypants/) por defecto. Para deshabilitarlo, establezca la propiedad `smartypants` a `false`:
```